components/style/properties/gecko.mako.rs

```diff
@@ -1964,6 +1965,21 @@ fn static_assert() {
         self.gecko.mTextEmphasisStyle = other.gecko.mTextEmphasisStyle;
     }
 
+    <%call expr="impl_color_setter('_webkit_text_fill_color', 'mWebkitTextFillColor')"></%call>
+    <%call expr="impl_color_copy('_webkit_text_fill_color', 'mWebkitTextFillColor')"></%call>
+    <%call expr="impl_color_clone('_webkit_text_fill_color', 'mWebkitTextFillColor')"></%call>
+
+    <%call expr="impl_color_setter('_webkit_text_stroke_color', 'mWebkitTextStrokeColor')"></%call>
+    <%call expr="impl_color_copy('_webkit_text_stroke_color', 'mWebkitTextStrokeColor')"></%call>
+    <%call expr="impl_color_clone('_webkit_text_stroke_color', 'mWebkitTextStrokeColor')"></%call>
+
+    #[allow(non_snake_case)]
+    pub fn set__webkit_text_stroke_width(&mut self, v: longhands::_webkit_text_stroke_width::computed_value::T) {
```

Comment:
You can probably do that instead of `_webkit_text_stroke_width` functions:
```
<% impl_app_units("_webkit_text_stroke_width", "mWebkitTextStrokeWidth", need_clone=False, round_to_pixels=True) %>
```